E-Legal/ViewControllers/SignUpController/SignUpViewController.swift

```diff
@@ -58,6 +54,32 @@ class SignUpViewController: UIViewController {
       signUpCell?.textFieldDateOfBirth.text = timeFormatter.stringFromDate(sender.date)
    }
 
+   func signUpNewuser() {
+    let newUser = User()
+    
+      newUser.username = signUpCell?.textFieldName.text
+      newUser.password = signUpCell?.textFieldPassword.text
+      newUser.email = signUpCell?.textFieldemail.text
+
+      // Sign up the user asynchronously
+      newUser.signUpInBackgroundWithBlock({ (succeed, error) -> Void in
+
+         if ((error) != nil) {
+            ApplicationHelper.showAlertView("Error", message: "\(error)", view: self)
+         } else {
+            ApplicationHelper.showAlertView("Success", message: "Signed Up", view: self)
+            dispatch_async(dispatch_get_main_queue(), { () -> Void in
+               print("Hello you are signed up")
```

Comment:
Remove prints on master we never commit prints unless we are debugging something